gnomad_qc/v4/variant_qc/random_forest.py

```diff
@@ -104,13 +99,30 @@ def train_rf(
             for x in test_intervals
         ]
 
+    if test:
+        logger.info("Filtering to chr22 and evaluation intervals for testing...")
+        chr22_interval = [hl.parse_locus_interval("chr22", reference_genome="GRCh38")]
+        ht = hl.filter_intervals(ht, chr22_interval + test_intervals)
+
+    logger.info("Annotating true positives and false positives in HT...")
+    fp_expr = ht.fail_hard_filters
+    tp_expr = ht.omni | ht.mills
+    if transmitted_singletons:
```

Comment:
switch of the logic from last time - I'll have to make a note to pass it in pass these in when I'm running the RF code in the future

Reply:
yeah, it's just personal preference, either way if fine, but I just like this way better